shared-modules/tesseract-3.05.01.json

```diff
@@ -0,0 +1,96 @@
+{
+    "name": "tesseract-bundle",
+    "no-autogen": true,
+    "sources": [
+        {
+            "type": "file",
+            "path": "noop-Makefile",
+            "dest-filename": "Makefile"
+        }
+    ],
+    "modules": [
+        {
+            "name": "libleptonica",
+            "buildsystem": "autotools",
+            "config-opts": [
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/DanBloomberg/leptonica/releases/download/1.74.4/leptonica-1.74.4.tar.gz",
+                    "sha256": "29c35426a416bf454413c6fec24c24a0b633e26144a17e98351b6dffaa4a833b"
+                }
+            ]
+        },
+        {
+            "name": "tesseract",
+            "buildsystem": "autotools",
+            "config-opts": [
+                "--disable-graphics"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/tesseract-ocr/tesseract/archive/3.05.01.tar.gz",
+                    "sha256": "05898f93c5d057fada49b9a116fc86ad9310ff1726a0f499c3e5211b3af47ec1"
+                }
+            ]
+        },
+        {
+            "name": "tessdata",
+            "no-autogen": true,
```

Comment:
You can use the `simple` buildsystem instead – although I'm curious what's going to happen if no `build-commands` is given while using it.

Reply:
It doesn't seem to be available in Flatpak 0.8.5 (I'm using Ubuntu 17.04 at the moment). I would prefer to keep it as is for now if possible.